src/Http/Authentication.Abstractions/src/IAuthenticationResultFeature.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// Used to store the AuthenticateResult for the HttpContext.
+    /// </summary>
+    public interface IAuthenticationResultFeature
+    {
+        /// <summary>
+        /// The <see cref="AuthenticateResult"/> for the request.
+        /// </summary>
+        AuthenticateResult Result { get; set; }
```

Comment:
I have a project that has multiple authentication scheme and no default scheme. So I don't even have `UseAuthentication` in the pipeline. Is it possible that the AuthenticateResult/Ticket/Properties for other scheme be available in this feature. If so I think this feature should be also set in Authorization middleware.

Reply:
You should consider create a virtual scheme that aggregates the multiple schemes (implements authenticate by calling authenticate on all the schemes and merges them), then could just use that as your default scheme